base/common/python/pki/__init__.py

```diff
@@ -582,10 +581,17 @@ def get(self, name):
             key = match.group(1)
             value = match.group(2)
 
-            if key.lower() == name.lower():
-                return value
+            if key.lower() != name.lower():
+                continue
 
-        return result
+            # remove quotes if any
+            if self.quote and value.startswith(self.quote) and value.endswith(self.quote):
+                length = len(self.quote)
+                value = value[length:-length]
```

Comment:
Do we need to accept a case where there is more than one quotation character, e.g. `""[application_version]""`?
If not, then this can simplify to `value = value[1:-1]` - and in the other places.

Reply:
Python does support triple quotes (`'''` or `"""`) but I guess we don't really need to support that. I've added a code to check the quote length and changed the quote removal like you described.